src/ui/Views/Organisation/Shared/ErrorSummary.cshtml

```diff
@@ -51,7 +51,8 @@
                 {
                     foreach (var error in state.Value.Errors)
                     {
-                        <li><a href="@Url.Action("About", "Organisation")#@state.Key-wrapper">@error.ErrorMessage</a></li>
+                        var action = Array.IndexOf(new string[] {"EmailAddress", "Telephone", "Url", "Addr1", "Addr2", "Addr3", "Addr4", "Postcode"}, state.Key) > -1 ? "Contact" : "About";
+                        <li><a href="@Url.Action(action, "Organisation")#@state.Key-wrapper">@error.ErrorMessage</a></li>
```

Comment:
How about my genius idea that I stole from you that you should put the list of keys into a static method on the specific views :-D